spek-samples/src/test/kotlin/org/jetbrains/spek/samples/subject/CalculatorTableSpec.kt

```diff
@@ -0,0 +1,38 @@
+package org.jetbrains.spek.samples.subject
+
+import org.jetbrains.samples.Calculator
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.describe
+import org.jetbrains.spek.api.dsl.it
+import org.jetbrains.spek.api.dsl.on
+import org.jetbrains.spek.table.example
+import org.jetbrains.spek.table.with
+import kotlin.test.assertEquals
+
+object CalculatorTableSpec : Spek({
+    describe("subtract") {
+        with(
+            example(4, 2),
```

Comment:
Adding the expected result in the data is an anti-pattern to me so I'd like not to encourage it (but I still want to allow it). Here is why : 
If you code the fizzBuzz kata this way, you'll end up with only one test with everything in it. Like this : 
```kotlin
forEvery(
   example(1, "1"),
   example(2, "2"),
   example(3, "fizz"),
   example(5, "buzz"),
   example(6, "fizz"),
   example(10, "buzz"),
   example(15, "fizzbuzz"),
   example(30, "fizzbuzz")
) { n, expected ->
   it("should work") {
       expect(fizzBuzz(n)) shouldBe expected
   }
}
```
It seems good at first but there is a very bad side effect : your documentation know sucks!
Instead, you can use it way better by fixing something (like the expected result) in your test. You'll get something like : 

```kotlin
forEvery(
   example(1),
   example(2)
) { n ->
   it("should echo the input") {
       expect(fizzBuzz(n)) shouldBe n.toString()
   }
}

forEvery(
   example(3),
   example(6)
   ) { n, expected ->
   it("should fizz") {
       expect(fizzBuzz(n)) shouldBe "fizz"
   }
}
.
.
.
```
Now the documentation is much nicer (and totally in the philosophy of spek).

Never the less, allowing the user to specify input *and* output in the exemple still necessary, because, sometimes, it still is the right way to do !
Let's use the roman numeral kata as on exemple : 
```kotlin

```kotlin
forEvery(
   example(1, "I"),
   example(5, "V"),
   example(10, "X"),
   example(50, "L")
) { n, expected ->
   it("should translate roman literals") {
       expect(toRoman(n)) shouldBe expected
   }
}
```